viewmaster/viewmaster.py

```
@@ -49,33 +77,81 @@
 #     LOG_ROOT_PREFIX_ = '/Library/WebServer/Logs/webapps/'
 ################################################################################
 
-from viewmaster_config import *
+from .viewmaster_config import *
```

Ruff (0.14.11)

`from .viewmaster_config import *` used; unable to detect undefined names

(F403)